apps/teams-test-app/src/App.tsx

```diff
@@ -16,29 +16,61 @@ const urlParams = new URLSearchParams(window.location.search);
 const getOriginsParam = urlParams.has('origins') && urlParams.get('origins') ? urlParams.get('origins') : '';
 const validMessageOrigins: string[] | undefined = getOriginsParam ? getOriginsParam.split(',') : undefined;
 
-// This is added for custom initialization when app can be initialized based upon a trigger/click.
-if (!urlParams.has('customInit') || !urlParams.get('customInit')) {
-  if (isTestBackCompat()) {
-    initialize(undefined, validMessageOrigins);
+if (!urlParams.has('testInit') || !urlParams.get('testInit')) {
```

Comment:
>!urlParams.has('testInit') || !urlParams.get('testInit') [](http://example.com/codeflow?start=4&length=56)

Isn't checking `urlParams.get('testInit') === null` sufficient?

Reply:
Logically it seems like it doesn't work if there is a url link looks as xxxxxx?testInit=false. And `urlParams.get('testInit') === null` has the same behavior if `testInit=true` or `testInit=false`